notification/twilio/validation.go

```diff
@@ -82,7 +82,7 @@ func validSID(n string) string {
 // Supported Country Codes
 // +1 = USA, +91 = India
```

Comment:
tiny nit: would you mind adding `+44 = United Kingdom` to comment ?

Reply:
@m17ch Thanks for picking that up. I've added it in